context/app/static/js/components/entity-search/Search/Search.jsx

```diff
@@ -13,9 +13,27 @@ import Pagination from 'js/components/entity-search/results/Pagination';
 
 import RequestTransporter from 'js/components/entity-search/searchkit-modifications/RequestTransporter';
 import Sidebar from 'js/components/entity-search/sidebar/Sidebar';
+import SearchBar from 'js/components/entity-search/SearchBar';
 import { SearchLayout, ResultsLayout } from './style';
 import { buildSortPairs, getRangeProps } from './utils';
 
+const query = new CustomQuery({
+  queryFn: (q) => {
+    return {
+      bool: {
+        must: [
+          {
+            simple_query_string: {
+              fields: ['all_text', 'description'],
```

Comment:
Particularly if we'll be revisiting the ES configuration, this might be something to keep an eye on. If memory serves, `all_text` is a field we create, but may be possible for ES to combine fields for search without actually duplicating them in the stored document.